ovnimp.go

```diff
@@ -334,9 +334,13 @@ func (odbi *ovndb) populateCache(updates libovsdb.TableUpdates) {
 func (odbi *ovndb) ConvertGoSetToStringArray(oset libovsdb.OvsSet) []string {
 	var ret = []string{}
 	for _, s := range oset.GoSet {
-		value, ok := s.(string)
-		if ok {
+		switch s.(type) {
+		case string:
+			value := s.(string)
 			ret = append(ret, value)
+		case libovsdb.UUID:
```

Comment:
Thanks for the improvement. Seems we should add at-least one UT to cover this so that uuids are also included as ConvertGoSetToStringArray is common module uses by all entities.

Reply:
@noah8713 i have added the Unit Testcase, so PTAL.